conda/notices/core.py

```diff
@@ -48,15 +48,29 @@ def display_notices(
         num_viewed_notices = len(viewed_notices)
 
     channel_notices = filter_notices(channel_notices, limit=limit, exclude=viewed_notices)
-    if len(channel_notices) == 0:
-        return
 
-    views.print_notices(channel_notices)
+    return ChannelNoticeSet(
+        channel_notices=channel_notices,
+        viewed_channel_notices=num_viewed_notices,
+        total_number_channel_notices=num_total_notices,
+    )
```

Comment:
Could we align the variable names here with the attribute names of ChannelNoticeSet?